src/scripts/main.js

```diff
@@ -23,6 +23,25 @@ const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
   // WRITE YOUR CODE HERE
+  const list = document.createElement('ul');
+
+  for (const key in data) {
+    if (Object.keys(data[key]).length !== 0) {
+      const listItem = document.createElement('li');
+
+      listItem.innerText = key;
+      list.append(listItem);
+      element.append(list);
+      createTree(listItem, data[key]);
+    } else {
+      const listItem = document.createElement('li');
+
+      listItem.innerText = key;
+      list.append(listItem);
+      element.append(list);
+      createTree(listItem, data[key]);
+    }
```

Comment:
why do you need `else` part if code is the same?
make it in the way when you don't need to repeat yourself